### PR TITLE
Forgot password

### PR DESCRIPTION
**Pull request by mjalbert99**

### CODE REVIEW

**Comment by mjalbert99** [comment]
Added the UI, app interactions, and firebase servers calls

File: app/src/main/java/com/example/activeamigo/ForgotPassword.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import com.google.android.gms.tasks.TaskCompletionSource;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.QuerySnapshot;
+
+public class ForgotPassword extends AppCompatActivity implements Alertable{
+   private FirebaseAuth auth;
+   private FirebaseFirestore db;
+   private EditText emailText;
```
Comment (cknguyen123) [inline app/src/main/java/com/example/activeamigo/ForgotPassword.java]: consistency convention: in login you set them null

File: app/src/main/java/com/example/activeamigo/ForgotPassword.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                     }
+                 }
+            });
+
+        });
+
+    }
+
+
+    // Check if the email already exists in the database
```
Comment (cknguyen123) [inline app/src/main/java/com/example/activeamigo/ForgotPassword.java]: consistency convention /** **/

Comment (mjalbert99) [reply app/src/main/java/com/example/activeamigo/ForgotPassword.java]: Changed format

File: app/src/main/java/com/example/activeamigo/ForgotPassword.java (hunk 1 above)
Comment (mjalbert99) [reply app/src/main/java/com/example/activeamigo/ForgotPassword.java]: Initialized to null

**Comment by mjalbert99** [comment]
added the tests

**Pull request merged**